test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1532,6 +1532,23 @@ def forward(self, x):
                       dynamic_axes={'input_1': [0, 1, 2],
                                     'output_1': [0, 1, 2]})
 
+    @disableScriptTest()
```

Comment:
If you are disbling scripting, why are you testing with if/else? This is not reflected in the model in tracing.